data/.gitignore

```diff
@@ -8,3 +8,4 @@ no-cache.txt
 update.php
 tos.html
 opml.xml
+applied_migrations.txt
```

Comment:
```suggestion
applied_migrations.txt
applied_migrations.txt.lock
```

Reply:
I cannot remember if gitignore would also match that one, but at least this way it is explicit